### ISSUE TITLE
`pwa-build.ts` not working?

### ISSUE THREAD

**Issue by userquin** [ctwhome/top-nuxt3#3]
I have it working, what's the problem here? I use `pnpm`.

![imagen](https://user-images.githubusercontent.com/6311119/147676424-ef6937a3-bd58-4741-a6c4-c21450b79c15.png)

**Comment by PMLS3** [comment]
@userquin how did you make it work?

**Comment by ctwhome** [comment]
thanks @userquin for the effort. I need to check it out better, It wasn't registering the manifest, so not showing the app name, icons etc. I am gonna have a look at it today

**Comment by PMLS3** [comment]
@ctwhome also got the same error... on built it doesnt take the css (windicss)

**Comment by PMLS3** [comment]
@ctwhome Did you manage to find a solution? Been trying the last couple of days but no success...

**Comment by ctwhome** [comment]
I can't find the problem either ☹️

**Comment by userquin** [comment]
@PMLS3 where are you deploying the app (vercel, netlify, custom server)? beware, the app needs a server middleware for some resources, just check the nuxt module added and maybe you cannot use it on your deployment.

Can you also provide some more input? some log and some screenshots for output build directory.

**Comment by userquin** [comment]
Right now, there is no static site generation available on nuxt3 so you must deploy the nuxt3 app and start the server module, I think.

As I mention on pwa plugin repo it is a POC and we should wait to nuxt team solution. It should work on local, but it is not ready for production (I mean to be deployed on production environment).

**Comment by userquin** [comment]
some more inputs: I built it using node 16.11.0 LTS

**Comment by ctwhome** [comment]
In my case, I just serve it on Netlify. I am just thinking if the middleware could be hosted as a cloud function on vercel or netlify. Thanks for the response!

**Comment by ctwhome** [comment]
I think we should close this issue and leave the comments on the readme of the template

**Comment by PMLS3** [comment]
> @PMLS3 where are you deploying the app (vercel, netlify, custom server)? beware, the app needs a server middleware for some resources, just check the nuxt module added and maybe you cannot use it on your deployment.
> 
> Can you also provide some more input? some log and some screenshots for output build directory.

I am deploying it to Google Cloud with Dockerbuild
![docker](https://user-images.githubusercontent.com/62140524/149758828-2936da84-fb18-4e6a-a420-a89fd269a4c6.PNG)

**Comment by userquin** [comment]
@PMLS3 uhmm, can you check the file is there? it seems the error comes from using the dynamic import to load the client manifest to add the pages to the sw precache manifets.

**Comment by userquin** [comment]
the build log cannot be downloaded

**Comment by PMLS3** [comment]
> @PMLS3
> 
> Be sure you have at least node v14.17.0 (I recommend at least 14.18.0), with node 16.11.0 LTS I have not that problem. Just check `node --version` on your deploy.
> 
> On your local is it working (build + serve)?

Yes local build + serve works 100%, trying to figure out where to see my build files on google cloud. Will let you know about the file as soon as I get it. Will try up my Dockerfile to use node v-16

**Comment by PMLS3** [comment]
> the build log cannot be downloaded

@ctwhome sorry, here is the txt
[build log.txt](https://github.com/ctwhome/top-nuxt3/files/7886094/build.log.txt)

**Comment by PMLS3** [comment]
@ctwhome Can you see if /usr/src/app/.output/server/chunks/client.manifest.mjs has build on yourside?

**Comment by ctwhome** [comment]
nope, nothing there
![image](https://user-images.githubusercontent.com/4195550/150412786-494b6f51-adfb-484b-9824-ff70051cbf73.png)

**Comment by PMLS3** [comment]
> nope, nothing there
> 
> ![image](https://user-images.githubusercontent.com/4195550/150412786-494b6f51-adfb-484b-9824-ff70051cbf73.png)
> 
> 

@userquin seems like it doesn't build with the file. Do you perhaps know why it doesn't. Locally it builds fine

**Comment by userquin** [comment]
@PMLS3 maybe you can ask on nuxt repo
